src/test/ui/nll/user-annotations/method-ufcs-inherent-1.rs

```diff
@@ -0,0 +1,19 @@
+#![feature(nll)]
+
+// Check that substitutions given on the self type (here, `A`) carry
+// through to NLL.
+
+struct A<'a> { x: &'a u32 }
+
+impl<'a> A<'a> {
+    fn new<'b, T>(x: &'a u32, y: T) -> Self {
+        Self { x }
+    }
+}
+
+fn foo<'a>() {
+    let v = 22;
+    let x = A::<'a>::new(&v, 22);
```

Comment:
I think you and I mirrored each other on our respective PR's; there should be an `//~ ERROR` on this line, right?

Reply:
Eep yes, will add.